exp/runtime/server/server.go

```diff
@@ -175,8 +175,8 @@ func (s *Server) validateHandler(handler ExtensionHandler) error {
 	}
 
 	// Get hook and handler request and response types.
-	hookRequestType := hookFuncType.In(0)  //nolint:ifshort
-	hookResponseType := hookFuncType.In(1) //nolint:ifshort
+	hookRequestType := hookFuncType.In(0)  
```

Comment:
`make lint-fix` made this change - I can remove it if we think it'll cause problems

Reply:
Kind of happy that I somehow never use it / have to use it :)